src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPProjectionProvider.cs

```diff
@@ -131,7 +131,9 @@ internal class DefaultLSPProjectionProvider : LSPProjectionProvider
             }
             else
             {
+#pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
Are we able to switch to the non-obsolete versions here? The only reason we need them is because WebTools calls directly into them, right?